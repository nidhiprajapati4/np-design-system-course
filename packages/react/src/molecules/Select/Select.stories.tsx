import React from "react";
import Select from "./Select";

import "@designsystem/scss/lib/Select.css";
const options = [
  {
    label: "Strict Black",
    value: "strict-black",
  },
  {
    label: "Heavenly Green",
    value: "heavenly-green",
  },
  {
    label: "Sweet Pink",
    value: "pink",
  },
];

export default {
  title: "Molecules/Select",
};

export const Common = () => <Select options={options} />;

export const CustomLabel = () => <Select label="select a color" options={options}/>
