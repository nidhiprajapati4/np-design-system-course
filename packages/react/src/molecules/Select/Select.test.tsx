import Select from "./Select";

test("renders all options passed to it", () => {
  expect(Select).toMatchSnapshot();
});
