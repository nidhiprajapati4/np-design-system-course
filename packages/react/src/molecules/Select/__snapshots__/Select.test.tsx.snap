// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders all options passed to it 1`] = `[Function]`;
